src/test/java/school/hei/haapi/integration/conf/TestUtils.java

```diff
@@ -105,6 +114,23 @@ public static void setUpCognito(CognitoComponent cognitoComponent) {
     when(cognitoComponent.getEmailByIdToken(MANAGER1_TOKEN)).thenReturn("test+manager1@hei.school");
   }
 
+  public static void setUpS3Service(S3Service s3Service, Student user) {
+    String USER_REF = user.getRef();
+    when(s3Service.getPresignedUrl(USER_REF, 180L)).thenReturn(USER_REF);
+    when(s3Service.uploadObjectToS3Bucket(USER_REF, getMockedFileAsByte("img", ".png")))
+        .thenReturn(USER_REF);
```

Comment:
use your user.getRef method here, no need to store it in a variable before using it once